kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractJKubeMojo.java

```diff
@@ -104,7 +104,7 @@ public abstract class AbstractJKubeMojo extends AbstractMojo implements KitLogge
     public void execute() throws MojoExecutionException, MojoFailureException {
         try {
             init();
-            if (!canExecute()) {
+            if (canExecute()) {
```

Comment:
The method name is not appropriate, or the log below is invalid. Method says that we 'canExecute' something, but log says we have skipped it

Reply:
You're right. We've created https://github.com/eclipse/jkube/issues/1510 in order to refactor this method's name